File: PHP/Car.php
<?php
require_once('account.php');
class Car {
    public $id = int;
    public $license = int;
    public $driver = string;
    public $passenger = string;

    public function __construct($license, $driver){
        $this->license = $license;
        $this->driver = $driver;
    }

    public function printDataCar() {
        echo "Licencia: $this->license 
              Driver:  {$this->driver->name}
              Numero de pasajeros: $this->passenger
              ";

    }

    public function getPassenger(){
        return $this->passenger;
    }

    public function setPassenger($passenger) {
        if($passenger == 4){
            $this->passenger = $passenger;
        } else {
            echo "Necesitas asignar 4 pasajeros";
        }
    }

}
?>

File: Java/Car.java
public class Car {
    private int id;
    private int passenger;
    private String license;
    private Account driver;
    protected int passengers;

    public Car(String license, Account driver) {
        this.license = license;
        this.driver = driver;

    }

    void printDataCar() {
        System.out.println("License: " + license + " Name Driver: " + driver.name + " Passenger: " 
        + passenger);
    }

    public int getPassenger() {
        return passenger;
    }

    public void setPassenger(int passenger) {
        if(passenger == 4) {
        this.passenger = passenger;
    } else {
        System.out.println("Necesitas asignar 4 pasajeros");
    }
    }




    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getLicense() {
        return license;
    }

    public void setLicense(String license) {
        this.license = license;
    }

    public Account getDriver() {
        return driver;
    }

    public void setDriver(Account driver) {
        this.driver = driver;
    }

    



}
